src/main/java/core/basesyntax/model/Operation.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.model;
+
+public enum Operation {
+    BALANCE("b"),
+    SUPPLY("s"),
+    PURCHASE("p"),
+    RETURN("r");
+
+    private String operation;
+
+    Operation(String operation) {
+        this.operation = operation;
+    }
+
+    public String getOperation() {
+        return operation;
+    }
+
+    public void setOperation(String operation) {
+        this.operation = operation;
+    }
+
+    public static Operation getOperationFromString(String operation) {
+        if (operation.equals("b")) {
+            return BALANCE;
+        }
+        if (operation.equals("s")) {
+            return SUPPLY;
+        }
+        if (operation.equals("p")) {
+            return PURCHASE;
+        }
+        if (operation.equals("r")) {
+            return RETURN;
+        }
+        return null;
+    }
```

Comment:
I think it's better to use loop here to find enum by String, or build some map. Otherwise, if we add a new type of operation, we will have to change this method. Let's try to implement more universal solutions